.github/workflows/build-and-test-arm.yaml

```diff
@@ -0,0 +1,33 @@
+name: build-and-test-arm
+
+on:
+  schedule:
+    - cron: 0 19 * * * # run at 4 AM JST
+  workflow_dispatch:
+
+jobs:
+  build-and-test-arm:
+    runs-on: ARM64
```

Comment:
Do we need an ARM64 job?
This repository is intended for use by Tier 4 projects, and we do not have a self-hosted runner for ARM64.

Reply:
I've pointed it out in #2 and it was removed.

Reply:
I see. Thank you!